src/NPacMan.Game/Game.cs

```diff
@@ -103,7 +103,7 @@ public void ChangeDirection(Direction direction)
 
         private void Tick(DateTime now)
         {
-            _gameStateMachine.RaiseEvent(_gameState, _gameStateMachine.Tick, new Tick { Now = now });
+            _gameStateMachine?.RaiseEvent(_gameState, _gameStateMachine.Tick, new Tick { Now = now });
```

Comment:
I think the actual problem is in the ctor when we set things up in the incorrect order 👍 _gameStateMachine is not nullable.